app/routes/meetings/components/MeetingDetail.js

```diff
@@ -123,6 +125,29 @@ class MeetingDetails extends Component<Props> {
     const canDelete = actionGrant && actionGrant.includes('delete');
     const canEdit = actionGrant && actionGrant.includes('edit');
 
+    const infoItems = [
+      {
+        key: 'Din status',
+        value: `${statusesText[statusMe]}`
+      },
+      {
+        key: 'Når',
+        value: <FromToTime from={meeting.startTime} to={meeting.endTime} />
+      },
+      {
+        key: 'Lokasjon',
```

Comment:
"Sted"? "Lokasjon" sounds kind of forced in Norwegian

Reply:
Agree. Just decided on not changing the word that was there originally. Will fix.